CHANGELOG.md

```diff
@@ -2,12 +2,42 @@
 
 ## Unreleased
 
+## 3.13.0 (Feature release)
+
+### New features
+
+#### Add a 'none' option and 'or' divider to checkboxes
+
+You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me.'
```

Comment:
'None of these options apply to me.' => 'None of these options apply to me'.

Reply:
```suggestion
You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me'.
```